zipline/finance/performance/period.py

```diff
@@ -212,6 +245,27 @@ def rollover(self):
             else:
                 del self._payout_last_sale_prices[asset]
 
+    def subdivide_period(self, capital_change):
+        self.calculate_performance()
+
+        # We are segmenting a previous subperiod that ran up until this point.
+        # Those performance stats are the total performance stats at this point
+        self._prev_subperiod_returns = self.total_returns
+        self._prev_subperiod_pnl = self.total_pnl
+        self._prev_subperiod_cash_flow = self.total_cash_flow
+
+        # Apply the capital change to the ending cash
+        self.ending_cash += capital_change
+
+        # We are segmenting a previous subperiod. Roll over the ending values
+        # of the previous into the starting values of the current. Reset cash
+        # flow
+        self._starting_value = self.ending_value
```

Comment:
https://github.com/quantopian/zipline/pull/1212/files#r64280750 applies here, though would this would be a different group of values.